.travis.yml

```diff
@@ -54,6 +56,8 @@ matrix:
         # Establish updated toolchain as default
         - sudo unlink /usr/bin/gcc && sudo ln -s /usr/bin/gcc-4.8 /usr/bin/gcc
         - sudo unlink /usr/bin/g++ && sudo ln -s /usr/bin/g++-4.8 /usr/bin/g++
+      script:
+        - ./bootstrap.sh -e -t && cd build  && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_RULE_MESSAGES=OFF .. && cmake --build . --parallel 3 && sudo make test ARGS="-j4 --output-on-failure" && make linter
```

Comment:
Why is it --parallel 3 here? It is 4 everywhere else.

Reply:
Same as below, just sloppy rebase/merge across machines and I haven't looked through the PR yet as my focus was travis doing better. I will fix this and everything else in hopes of getting a full green in less time. Thanks for helping!!

Reply:
Thank you for doing all of this, it was really needed!